pymc/distributions/multivariate.py

```diff
@@ -530,10 +530,18 @@ class Multinomial(Discrete):
 
     @classmethod
     def dist(cls, n, p, *args, **kwargs):
-        p = p / at.sum(p, axis=-1, keepdims=True)
+        if isinstance(p, np.ndarray) or isinstance(p, list):
+            if (np.array(p) < 0).any():
+                raise ValueError("Negative probabilities are not valid")
+            p_sum = np.sum([p], axis=-1)
+            if (p_sum != 1.0).any(): 
```

Comment:
Same here

Reply:
Also in the `multinomial.logp` we should use `at.isclose` instead of `at.eq`. That is probably why one of the tests is failing

Reply:
I have made these changes now.